### DISCUSSION TITLE
Installation/setup

### DISCUSSION THREAD

**Discussion by Dan-McCalley** [muscbridge/PyDKE#253] [Technical Support] [answered]
Hi all, 

Followed the steps on the wiki for download & setup, but still having trouble running a basic implementation of pydesigner.

`conda activate dmri`
`(dmri) dan@Dans-science-macs-MacBook-Pro dwi % pydesigner -s -rpe_pairs 1 --verbose -o /Users/dan/Desktop/Diffusion_testing/dwi/sub-1202_diffusion_test /Users/dan/Desktop/Diffusion_testing/dwi/sub-1202_ses-v1_dwi.nii.gz, /Users/dan/Desktop/Diffusion_testing/dwi/sub-1202_ses-v1_dir-PA_dwi.nii.gz`
`zsh: command not found: pydesigner `

Any suggestions on what might have gone wrong during setup? Guessing if pydesigner were installed correctly commandline output would say something other than command not found

Let me know what info you need from me!

Thanks!

**Comment by TheJaeger** [comment, marked as answer]
Hi @Dan-McCalley,

It appears that your PyDesigner installation didn't complete. Make sure your installation finishes to completion. The exact commands for installing PyDesigner can be found at https://pydesigner.readthedocs.io/en/latest/installation/pydesigner.html

Remember to activate your `dmri` environment before installing PyD.

**Reply by Dan-McCalley** [reply]
Hmm, re-went through the installation process. Looks like the issue is occuring during the pip instal process. 

Attached the command line output here. Double checked and all my other dependencies (FSL, python, & mrtrix) are up to date
[CL_Error_PyDesigner.txt](https://github.com/m-ama/PyDesigner/files/6207294/CL_Error_PyDesigner.txt)

**Reply by TheJaeger** [reply]
Refer to [my post](https://github.com/m-ama/PyDesigner/issues/242#issuecomment-773714922) on issue #242 for fix. This is a known bug due to conflicting package dependencies.

**Reply by Dan-McCalley** [reply]
Awesome, problem solved! Thanks!
